docs/docs/porting-an-html-site-to-gatsby.md

```diff
@@ -0,0 +1,543 @@
+import Helmet from "react-helmet"
+
+export default ({ children }) => (
+  <>
+    <Helmet></Helmet>
+    <header></header>
+    <main>{children}</main>
+  </>
+)
+```
+
+The common elements between the `/index.html` and `/who/index.html` files can now copied from `/src/index.js` into the `<Layout>` component. A second prop is also added here and used in a JavaScript expression in the `<title>` element. The new expression results in a dynamic version of the title, depending on what is passed as the `breadcrumbs` prop. If the `breadcrumbs` prop is provided, it is joined and added to the end of the base title, `Taylor's Tidy Trees`. You'll see the prop in use later on when porting the `/who/index.html` page.
+
+```jsx:title=/gatsby-site/src/components/Layout.js
+import React from "react"
+import Helmet from "react-helmet"
+import { Link } from "gatsby"
+
+import "../styles/normalize.css"
+import "../styles/style.css"
+
+export default ({ children, breadcrumbs }) => (
+  <>
+    <Helmet>
+      <title>
+        Taylor's Tidy Trees
+        {breadcrumbs ? ` - ${breadcrumbs.join(" - ")}` : ``}
+      </title>
+      <link href="/favicon.ico" rel="shortcut icon" type="image/x-icon" />
+    </Helmet>
+    <header>
+      <Link to="/" className="brand-color logo-text">
+        Taylor's Tidy Trees
+      </Link>
+      <nav>
+        <ul>
+          <li>
+            <Link to="/about.html">About</Link>
+          </li>
+          <li>
+            <Link to="/services/index.html">Services</Link>
+          </li>
+          <li>
+            <Link to="/who/index.html">Who We Are</Link>
+          </li>
+          <li>
+            <Link to="/contact.html">Contact</Link>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>{children}</main>
+  </>
+)
+```
+
+You can now use the `<Layout>` component to create a `/src/who/index.js` page file:
+
+```jsx:title=/gatsby-site/src/pages/who/index.js
+import React from "react"
+import Layout from "../components/Layout"
+import { Link } from "gatsby"
+
+export default () => (
+  <Layout breadcrumbs={["Who We Are"]}>
+    <h1>Who We Are</h1>
+    <h2>These are our staff:</h2>
+    <ul>
+      <li>
+        <Link to="/who/ella-arborist">Ella (Arborist)</Link>
+      </li>
+      <li>
+        <Link to="/who/sam-surgeon">Sam (Tree Surgeon)</Link>
+      </li>
+      <li>
+        <Link to="/who/marin-leafer">Marin (Leafer)</Link>
+      </li>
+    </ul>
+  </Layout>
+)
+```
+
+The `Who We Are` link in `index.js` should now work! Now use the `<Layout>` component in the `index.js` page file too:
+
+```jsx:title=/gatsby-site/src/pages/index.js
+import React from "react"
+import Layout from "../components/Layout" // highlight-line
+
+export default () => (
+  {/* highlight-start */}
+  <Layout>
+    <h1>Welcome To Taylor's Tidy Trees</h1>
+    <h2>We care about trees of all kinds!</h2>
+  </Layout>
+  {/* highlight-end */}
+)
+```
+
+Have a check that the `Who We Are` link is still working.
```

Comment:
If it isn't, do you have any advice for what they should do next to investigate?

Reply:
Good point, I think the only way it'd go wrong at that point is if elements the Layout is replacing haven't been completely removed:
```suggestion
Have a check that the `Who We Are` link is still working. If not, check all the elements being replaced by the `<Layout>` component has been removed.
```

That might be clear enough with the example above though, maybe the line could just be deleted. What do you think?

Reply:
Decided to updated with a note to check with the code snippet directly above the line:

>Have a check that the `Who We Are` link is still working. If not, check that the content is wrapped correctly with the `<Layout>` component as shown above.